driver/src/main/com/mongodb/util/JSONSerializers.java

```diff
@@ -104,6 +105,7 @@ static ClassMapBasedObjectSerializer addCommonSerializers() {
         serializer.addObjectSerializer(ObjectId.class, new ObjectIdSerializer(serializer));
         serializer.addObjectSerializer(Pattern.class, new PatternSerializer(serializer));
         serializer.addObjectSerializer(String.class, new StringSerializer());
+        serializer.addObjectSerializer(Symbol.class, new SymbolSerializer(serializer));
```

Comment:
Will add a new Jira - I had rolled it up as the toString() method blows up when using Symbol.